pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1273,6 +1273,11 @@ self: super: {
   # Support recent versions of fast-logger.
   spacecookie = doJailbreak super.spacecookie;
 
+  # 2020-02-28 this was broken and is needed by glirc -- Kiwi
+  hookup = doJailbreak (super.hookup.override {
+    network = self.network_3_0_1_1;
+  });
```

Comment:
Could you file an issue upstream about hookup not being able to be compiled with a later version of network, and put a link to that here?

This is so that we can know when it is possible to drop this `network` override.

Reply:
Yes, please, report this issue upstream! :+1:

Reply:
i've filed it on the most urgent of issue trackers. i.e. i messaged glguy on irc.

Reply:
@Kiwi We want you to actually file an issue on the upstream issue tracker so that you can link directly to it in this comment.

That way, when someone else comes along and wonders whether a new release of `hookup` has been made with this change, they can just visit the issue and see whether or not it is closed.  (If it is closed, and a new release of `hookup` has been made since then, then they can probably go ahead and remove the override.)

Reply:
glguy usually fixes things as soon as i ask him :heart: we might not even need to merge this (I didn't realize hookup was actually part of irc-core before otherwise I could have tried that first)

Reply:
> glguy usually fixes things as soon as i ask him

Ah, okay, so do you recommend we wait until a new version of hookup has been released?

Reply:
yeah i'd wait for now. at least until i hear back from him if there's a reason he can't yet. but it's likely we can save a commit/merge/noise/space

Reply:
> yeah i'd wait for now. at least until i hear back from him if there's a reason he can't yet. but it's likely we can save a commit/merge/noise/space

glguy is updating version bounds and doing a new release today which should solve this without this PR. :D